libs/utils/utils.js

```diff
@@ -568,3 +501,218 @@ export function createIntersectionObserver({ el, callback, once = true, options
+                if (ptax.isUFT) visibleTopics.push(parent);
+              }
+            });
+          }
+        }
+      } else {
+        // eslint-disable-next-line no-console
+        console.warn(`Unknown topic in tags list: ${tag} ${path ? `on page ${path}` : '(current page)'}`);
+      }
+    });
+    return { category, topics, visibleTopics, allTopics };
+  }
+  return { category, topics };
+}
+
+export async function loadTaxonomy() {
+  taxonomyModule = await taxonomyLibrary.default(getPrefix());
+  if (taxonomyModule) {
+    // taxonomy loaded, post loading adjustments
+    // fix the links which have been created before the taxonomy has been loaded
+    // (pre lcp or in lcp block).
+    document.querySelectorAll('[data-topic-link]').forEach((a) => {
+      const topic = a.dataset.topicLink;
+      const tax = taxonomyModule.get(topic);
+      if (tax) {
+        a.href = tax.link;
+      } else {
+        // eslint-disable-next-line no-console
+        console.warn(`Trying to get a link for an unknown topic: ${topic} (current page)`);
+        a.href = '#';
+      }
+      delete a.dataset.topicLink;
+    });
+
+    // adjust meta article:tag
+
+    const currentTags = getMetadata('article:tag', true) || [];
+    const articleTax = computeTaxonomyFromTopics(taxonomyModule, currentTags);
+
+    const allTopics = articleTax.allTopics || [];
+    allTopics.forEach((topic) => {
+      if (!currentTags.includes(topic)) {
+        // computed topic (parent...) is not in meta -> add it
+        const newMetaTag = document.createElement('meta');
+        newMetaTag.setAttribute('property', 'article:tag');
+        newMetaTag.setAttribute('content', topic);
+        document.head.append(newMetaTag);
+      }
+    });
+
+    currentTags.forEach((tag) => {
+      const tax = taxonomyModule.get(tag);
+      if (tax && tax.skipMeta) {
+        // if skipMeta, remove from meta "article:tag"
+        const meta = document.querySelector(`[property="article:tag"][content="${tag}"]`);
+        if (meta) {
+          meta.remove();
+        }
+        // but add as meta with name
+        const newMetaTag = document.createElement('meta');
+        newMetaTag.setAttribute('name', tag);
+        newMetaTag.setAttribute('content', 'true');
+        document.head.append(newMetaTag);
+      }
+    });
+  }
+}
+
+/**
+* Load the Privacy library
+*/
+function loadPrivacy() {
+  // Configure Privacy
+  window.fedsConfig = {
+    privacy: {
+      otDomainId: '7a5eb705-95ed-4cc4-a11d-0cc5760e93db',
+      footerLinkSelector: '[href="https://www.adobe.com/#openPrivacy"]',
+    },
+  };
+
+  const env = getConfig().env.name === 'prod' ? '' : 'stage.';
+  loadScript(`https://www.${env}adobe.com/etc.clientlibs/globalnav/clientlibs/base/privacy-standalone.js`);
+}
+
+export async function loadArea(area = document) {
+  const config = getConfig();
+  const isDoc = area === document;
+
+  if (isDoc) {
+    decorateHeader();
+  }
+
+  await loadTaxonomy();
```

Comment:
Remove. Also: this would load taxonomy for every fragment as well since it's not being checked in the doc.